AEPServices/Sources/ui/fullscreen/FullscreenMessage.swift

```diff
@@ -195,7 +216,9 @@ public class FullscreenMessage: NSObject, WKNavigationDelegate, FullscreenPresen
                 self.webView?.frame = newFrame
             }) { _ in
                 self.webView?.removeFromSuperview()
-                self.webView = nil
+                if shouldDeallocateWebView {
```

Comment:
Is this complexity worth it? Why not just create a new web view each time you need to show it?

Reply:
From Assurance point of view, it was worth it. Especially with `AssuranceStatusUI` which is supposed to appear and reappear as the user taps the `AssuranceFloatingbutton`

It saved me from saving the state of the StatusUI
And also reusing webView minimized the number of javascript commands I have to run every time when it kicks off.. or to load the webview to the current state

Also thinking beyond, the `hide` feature opens up a flexible usage of FullScreenMessage... Example(rendering videos / filling up online form).. where getting rid of the fullscreen message temporarily without losing the current state will be a bonus